.circleci/config.yml

```diff
@@ -181,10 +181,12 @@ jobs:
           steps:
             - run:
                 name: Make a pseudo-tag for the nightly build
-                command: git tag nightly-$(date +%Y-%m-%d)
+                command: |
+                  latest_tag=$(git describe --tags --abbrev=0)
+                  git tag ${latest_tag}+nightly-$(date +%Y-%m-%d)
```

Comment:
Based on [this document](http://www.fifi.org/doc/debian-policy/policy.html/ch-versions.html) I should probably move away from using `-` as the date separator, as the last one will get interpreted as the debian revision.